astropy/table/table.py

```diff
@@ -809,12 +809,13 @@ def _base_repr_(self, html=False, descr_vals=None, max_width=None,
                 descr_vals.append('masked=True')
             descr_vals.append('length={0}'.format(len(self)))
 
-        descr = '<' + ' '.join(descr_vals) + '>\n'
-
+        descr = ' '.join(descr_vals)
         if html:
             from ..utils.xml.writer import xml_escape
-            descr = xml_escape(descr)
-
+            descr = '<i>{0}</i>\n'.format(xml_escape(descr))
+        else:
+            descr = '<{0}>\n'.format(descr)
+            
```

Comment:
Please remove the extra whitespace from this line (as it triggers the style checker to fail).